src/uct/ib/base/ib_iface.c

```diff
@@ -1032,7 +1032,13 @@ ucs_status_t uct_ib_verbs_create_cq(uct_ib_iface_t *iface, uct_ib_dir_t dir,
     }
 
     if (!cq) {
-        ucs_error("ibv_create_cq(cqe=%d) failed: %m", cq_size);
+        UCS_STRING_BUFFER_ONSTACK(msg, 256);
```

Comment:
pls, move it to the beginning of the function in the place where we declare local variables and assign some value to them